training/nvidia-bootc/Containerfile

```diff
@@ -187,7 +187,11 @@ fi
 # Also make sure not to duplicate if a base image already has it specified.
 RUN grep -q /usr/lib/containers/storage /etc/containers/storage.conf || \
     sed -i -e '/additionalimage.*/a "/usr/lib/containers/storage",' \
-	/etc/containers/storage.conf
+	/etc/containers/storage.conf && \
+    sed -i -e 's/^# force_mask.*$/force_mask = "shared"/' \
+    /etc/containers/storage.conf && \
+    sed -i -e 's@^#mount_program = .*@mount_program = "/usr/bin/fuse-overlayfs"@' \
```

Comment:
But won't this force fuse-overlayfs for *rootful* containers stored in `/var/lib/containers` too? That seems quite suboptimal.

Reply:
agree it should be just storage options on the command line (I believe that is what he is doing in the current approach he is trying - more recent than this PR)

Reply:
Pushed modified code

Reply:
If you chown the file system, it will require fuse-overlayfs in order to see the intended user permission flags.

I am still of the opinion that this is overkill and not needed until we fully understand the UI intended for RHEL AI.

If only one user is going to use this system, then going though these hoops rather then just `sudo ilab` is just a risky overhead.

Reply:
Yeah the more we progress go the more I realize how far from trivial this is the more I feel like giving up and going sudo